src/compiler/core/build.gradle.kts

```diff
@@ -16,6 +16,7 @@ kotlin {
     linuxX64()
     js(IR) {
         nodejs()
+        useEsModules()
```

Comment:
Is that needed for the core? I thought this was only needed for the npm module on which the playground depends.